src/lyapunovs.jl

```diff
@@ -42,11 +42,9 @@ function lyapunovs(ds::DiscreteDS, N::Real; Ttr::Int= 100)
     J = jac(u)
     K = J*Q
 
-    F = qrfact(K)
-    Q .= F[:Q]
-    A_mul_signB!(Q, F[:R]) #ensure QR gives positive diagonal to R
```

Comment:
Cool but why is this line missing from the new code? As far as I know it is necessary for the algorithm to work...?
The line: `A_mul_signB!(Q, F[:R]) #ensure QR gives positive diagonal to R`